src/plus/integrations/authentication/integrationAuthentication.ts

```diff
@@ -94,27 +100,58 @@ abstract class IntegrationAuthenticationProviderBase<ID extends IntegrationId =
 
 	protected abstract restoreSession(sessionId: string): Promise<ProviderAuthenticationSession | undefined>;
 
-	protected async deleteSecret(key: IntegrationAuthenticationKeys) {
+	protected async deleteSecret(key: IntegrationAuthenticationKeys, sessionId: string) {
 		await this.container.storage.deleteSecret(key);
+		await this.authenticationService.removeConfigured({
+			integrationId: this.authProviderId,
+			domain: isSelfHostedIntegrationId(this.authProviderId) ? sessionId : undefined,
+		});
```

Comment:
Nit for another day, but it feels like these operations should be combined into the AuthService rather than in the base class.